notebooks/3.jump-analysis/nbconverted/3.jump_analysis.py

```diff
@@ -93,6 +96,13 @@
 shared_feature_space = load_json_file(shared_feature_space_path)
 shared_features = shared_feature_space["features"]
 
+# Experimenmta metadata
+jump_exp_meta = pd.read_csv(
+    "https://raw.githubusercontent.com/WayScience/JUMP-single-cell/main/reference_plate_data/experiment-metadata.tsv",
```

Comment:
protip: you can press the `y` key when you're navigated to the file to extract the specific version controlled commit. You should use versioned links (rather than `main`)

i.e., simply replace `main` with `d00868cfdb18143db0469e3cc0700b35c03cf811` to access the version controlled file (otherwise, subsequent updates to this file could break functionality)